src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[];
+  status: string;
+  handleClick: (event: React.MouseEvent) => void;
+  deleteTodos: (todos: number[], isInUpdate: boolean) => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  status,
+  handleClick,
+  deleteTodos,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {todos?.filter(td => td.completed !== true).length} items left
```

Comment:
Move this logic to the helper variable and use it here
```suggestion
        {todos?.filter(td => !td.completed).length} items left
```